server/models.py

```diff
@@ -1682,6 +1683,10 @@ def mimetype(self):
             return 'application/octet-stream'
         return guess[0]
 
+    @property
+    def download_link(self):
+        return '/files/{}'.format(encode_id(self.id))
```

Comment:
This is because you are bot always in a context where you can use `url_for` (mainly Jobs). I'd rather update it one place than in all of the jobs.